crates/hdk/src/hash_path.rs

```diff
@@ -10,9 +10,9 @@
 /// - The second level is optional as `Option<String>`
 pub mod anchor;
 
-/// The generic [ `path::Path` ] pattern.
+/// The generic [Path](hdi::prelude::Path) pattern.
```

Comment:
These docs don't really belong here any more, `Path` was moved to the HDI and the docs are duplicated there